lib/developer_portal/app/controllers/developer_portal/admin/applications/keys_controller.rb

```diff
@@ -13,7 +13,7 @@ def new
   end
 
   def create
-    @key = @cinstance.application_keys.add(params[:key])
+    @key = @cinstance.application_keys.add(params.permit!.to_h[:key])
```

Comment:
Why `permit!` everything here?
Isn't `key` required?

Or if it is not required, then would that work better?

```suggestion
    @key = @cinstance.application_keys.add(params.permit(:key)[:key])
```